src/server/pps/server/master.go

```diff
@@ -351,7 +315,7 @@ func (a *apiServer) upsertWorkersForPipeline(pipelineInfo *pps.PipelineInfo) err
 		options := a.getWorkerOptions(
 			pipelineInfo.Pipeline.Name,
 			pipelineInfo.Version,
-			int32(parallelism),
+			0,
```

Comment:
Is this supposed to be hard coded to zero here? It looks like this is always going to be the value passed to the replication controller for the number of replicas?